backend/src/controllers/boards/list-boards.js

```diff
@@ -0,0 +1,17 @@
+import { connection as knex } from '../../database/connection.js';
+
+const listBoards = async (req, res)=>{
+	const favorited = req.query.favorited;
+	try{
+		if(favorited){
+			const boards = await knex('boards').where('favorited', true);
+			return res.status(200).json(boards);
+		}
+		const boards = await knex('boards');
+		return res.status(200).json(boards);
```

Comment:
Tava fazendo uns testes aqui pra ver o retorno dessa listagem de favoritos, e aqui tbm o retorno da data traz fuso horário... pelo que vi, o retorno pro cliente (mesmo consultando no banco de dados) sempre traz o fuso horário se não for formatado... vamos ter que fazer isso em todas as rotas de retorno de data.
Aí fiz o que a gente tinha feito em criação e edição de board e trouxe a data certinha, segue o código:
import format from 'date-fns/format';

 const formattedBoards = boards.map(board => ({
            ...board,
            creation_date: format(new Date(board.creation_date), 'yyyy-MM-dd kk:mm:ss'),
            update_date: format(new Date(board.update_date), 'yyyy-MM-dd kk:mm:ss')
        }));
        return res.status(200).json(formattedBoards);